MainModule/Server/Core/Functions.lua

```diff
@@ -1243,7 +1243,7 @@ return function(Vargs, GetEnv)
 
 				new.Name = player.Name
 				new.Archivable = false
-				new.HumanoidRootPart.CFrame = pos*CFrame.Angles(0, math.rad((360/num)*i), 0) * CFrame.new((num*0.2)+5, 0, 0)
+				new:PivotTo(pos * CFrame.Angles(0, math.rad((360/num)*i+90), 0) * CFrame.new((num*0.2)+5, 0, 0) * CFrame.Angles(0, math.pi / 2, 0))
```

Comment:
Idk, I hear PivotTo can be pretty janky at times

Reply:
What?

Reply:
It means PivotTo can be pretty jaggedy when moving compared to setting CFrame

Reply:
> It means PivotTo can be pretty jaggedy when moving compared to setting CFrame

I don't think that would cause issues here though. This isn't some tween mover. It only positions the characters once